### PR TITLE
virsh.freepages: Fix fail when node without memory

### PR DESCRIPTION
**Pull request by waynesun09**
For some ppc machines, not all nodes are provided with memory,
freepages command will fail when with '--all' in such situation,
which I think is acceptable.

Fix this fail situation by update status_error value.

Signed-off-by: Wayne Sun <gsun@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/host/virsh_freepages.py [hunk 1]
```diff
@@ -124,6 +126,11 @@ def run(test, params, env):
                                      pagesize=page,
                                      options=option,
                                      debug=True)
+
+            # the node without memory will fail command and is expected
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/host/virsh_freepages.py]: the node without memory will fail and it is expected

**Comment by waynesun09** [comment]
@balamuruhans updated, thx

**Comment by balamuruhans** [comment]
ACK

**Comment by Hao-Liu** [comment]
Thanks @waynesun09 @balamuruhans, Merging this.

**Pull request merged**